### ISSUE TITLE
Chinese character dropped from tex output

### ISSUE THREAD

**Issue by jkr** [jgm/texmath#110]
Moved from https://github.com/jgm/pandoc/issues/4243

TeX output drops Chinese character. I assume the problem is in `Text.TeXMath.Unicode.ToTex`.  Not sure what the proper behavior should be, since cjk TeX is so thorny, but since TeX is what is represented in markdown (and so should ideally have a one-to-one relationship with native) it probably shouldn't be silently dropped.

omath.xml :

~~~xml
<m:oMath>
  <m:r>
    <m:rPr>
      <m:nor/>
    </m:rPr>
    <m:t>或</m:t>
  </m:r>
</m:oMath>
~~~

~~~
$ texmath -f omml -t mathml omath.xml
<math display="block" xmlns="http://www.w3.org/1998/Math/MathML">
  <mstyle mathvariant="normal">
    <mi>或</mi>
  </mstyle>
</math>
~~~
~~~
$ texmath -f omml -t tex omath.xml   
\mathrm{}
~~~

**Comment by jgm** [comment]
Looks like `fallback` is being triggered in `charToString`.  This uses `getASCII`, and of course there's no ascii equivalent to the Chinese character.

**Comment by jgm** [comment]
Fixed by v 0.11.
